grails-app/views/order/list.gsp

```diff
@@ -101,29 +103,28 @@
 											<g:set var="lineItems" value="${orderInstance?.orderItems?.findAll { it.orderItemStatusCode != OrderItemStatusCode.CANCELED }}"/>
 											${lineItems.size()?:0}
 										</td>
+										<td class="right middle">
+											<g:formatNumber number="${orderInstance.totalPrice()}"/>
+											${orderInstance.currencyCode?:grailsApplication.config.openboxes.locale.defaultCurrencyCode}
+										</td>
 									</tr>
 								</g:each>
 							</tbody>
-                            <%--
-							<tfoot>
-							<tr class="odd">
-								<th colspan="7"><label>${warehouse.message(code:'default.total.label')}</label></th>
-								<th colspan="1" class="right">
-									<div class="text large">
-
-										<g:formatNumber number="${totalPrice}"/>
-										${grailsApplication.config.openboxes.locale.defaultCurrencyCode}
-
-									</div>
-								</th>
-								<th>
-
-								</th>
-							</tr>
-							</tfoot>
-							--%>
+%{--							<tfoot>--}%
```

Comment:
Do we need this leftover?

Reply:
I want to add this back in once we get the currency conversion part done. Right now all of the orders have a total amount, but we cannot add them all together because the currencies might be different.